src/clj/puppetlabs/services/request_handler/request_handler_service.clj

```diff
@@ -2,14 +2,8 @@
   (:require [puppetlabs.trapperkeeper.core :as tk]
             [puppetlabs.services.protocols.request-handler :as handler]
             [puppetlabs.services.request-handler.request-handler-core :as core]
-            [puppetlabs.services.jruby.jruby-puppet-service :as jruby]
             [puppetlabs.trapperkeeper.services :as tk-services]))
 
-(defn- handle-request
-  [request jruby-service config]
-  (jruby/with-jruby-puppet jruby-puppet jruby-service
-    (core/handle-request request jruby-puppet config)))
-
```

Comment:
Yeah, @dankreek and I talked about that and waffled a bit.

I had found it a bit odd to be doing more seemingly domain specific logic like getting a jruby-puppet instance and possibly construct a ring response from the service namespace rather than core, which I kind of thought was where @cprice404 was on this as well.

I don't have a strong opinion on this, though - other than that I want to get this PR in soon so we can move on.